### DISCUSSION TITLE
404 Error

### DISCUSSION THREAD

**Discussion by educationalprogram** [community/community#31010] [Other Feature Feedback, Questions, & Ideas]
Hi! 
I created a repository with an html site, uploaded all the files there and published it according to the instructions (https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages). 
Gave 404 error when going to https://educationalprogram.github.io
I deleted the repository and recreated it, repeating the entire sequence of actions. The result is a 404 error. 
What is the problem? Why is my page not being published?

**Comment by ihnatenkoo** [comment]
Hello, the name of your repository can be educationalprogram.github.io
Rename it, and then you can view the site in link https://educationalprogram.github.io

**Reply by educationalprogram** [reply]
If you follow this link, you will find a 404 error.

**Reply by ihnatenkoo** [reply]
> If you follow this link, you will find a 404 error.

Now your repository has name "main", rename it to "educationalprogram.github.io"

**Reply by educationalprogram** [reply]
> > If you follow this link, you will find a 404 error.
> 
> Now your repository has name "main", rename it to "educationalprogram.github.io"

I'll try, but that doesn't seem to be the cause of the error.

**Reply by ihnatenkoo** [reply]
> > > If you follow this link, you will find a 404 error.
> > 
> > 
> > Now your repository has name "main", rename it to "educationalprogram.github.io"
> 
> I'll try, but that doesn't seem to be the cause of the error.

[https://educationalprogram.github.io/index.html](https://educationalprogram.github.io/index.html)
try it, in your site configured redirect?

**Reply by educationalprogram** [reply]
> > > > If you follow this link, you will find a 404 error.
> > > 
> > > 
> > > Now your repository has name "main", rename it to "educationalprogram.github.io"
> > 
> > 
> > I'll try, but that doesn't seem to be the cause of the error.
> 
> https://educationalprogram.github.io/index.html try it, in your site configured redirect?

Still 404. DNS record configured correctly.
